expenses-manager-backend/src/main/java/com/endava/expensesmanager/entity/Expense.java

```diff
@@ -0,0 +1,35 @@
+package com.endava.expensesmanager.entity;
+
+import jakarta.persistence.*;
+import lombok.Data;
+
+import java.sql.Timestamp;
+
+@Entity
+@Data
+public class Expense{
+    @Id
+    @Column(name = "expense_id")
+    private String expenseId;
+
+    @Column(name = "expense_description", nullable = true, unique = false)
+    private String description;
+
+    @Column(name = "expense_date", nullable = false, unique = false)
+    private Timestamp expenseDate;
```

Comment:
Hai sa incercam sa folosim `LocalDate` sau `LocalDateTime` in java - vezi cum se poate serializa/deserializa data/timestamp din sql in obiecte de java